pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -91,6 +92,12 @@ const (
 
 	InfraGarbageCollectorControllerServiceAccountName = "garbage-collector-controller"
 	GarbageCollectorControllerRoleName                = "system:garbage-collector-controller"
+
+	InfraTemplateInstanceControllerServiceAccountName = "template-instance-controller"
```

Comment:
if https://github.com/openshift/origin/pull/14150 merges ahead of you, I'd like to use the new pattern.  It aligns us with usptream and allows us to start splitting out ownership of launch mechanism and config information.